### PR TITLE
fix: add itself13

### PR DESCRIPTION
**Pull request by itself13**
here is my contribution

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/677"></a>

### CODE REVIEW

File: public/data/itself13.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+        "url": "https://twitter.com/dx_singhal",
+        "icon": "twitter"
+      },
+      {
+        "name": "Follow me on Instagram",
+        "url": "https://www.instagram.com/ii_am_ayuuu/",
+        "icon": "Instagram"
+      }
+    ]
+  }
```
Comment (Pradumnasaraf) [inline public/data/itself13.json:22]:
```suggestion
  }
  
```
### Just add a single line below 22 (With no in Spaces in that).

**Review by Pradumnasaraf — CHANGES_REQUESTED** [review]

**Comment by Pradumnasaraf** [comment]
**+** It will be great if you change the title of the PR from `hello` to something like `Profile Addition`.

**Review by schmelto — APPROVED** [review]
image

File: public/data/itself13.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
```
Comment (krishguptadev) [inline public/data/itself13.json:23]:
```suggestion

```

File: public/data/itself13.json (hunk 1 above)
Comment (krishguptadev) [inline public/data/itself13.json:22]:
```suggestion
}
```

Comment (krishguptadev) [reply public/data/itself13.json:22]: This is a wrong suggestion @Pradumnasaraf, please check my comments below...

File: public/data/itself13.json [hunk 3]
```diff
@@ -0,0 +1,23 @@
+{
+    "name": "Ayush Singhal",
+    "bio": "MERN stack developer",
+    "avatar": "https://github.com/itself13.png",
+    "links": [
+      {
+        "name": "Follow me on GitHub",
```
Comment (krishguptadev) [inline public/data/itself13.json:7]:
```suggestion
        "name": "Check out my GitHub",
```

File: public/data/itself13.json [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+        "url": "https://github.com/itself13",
+        "icon": "github"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/dx_singhal",
+        "icon": "twitter"
+      },
+      {
+        "name": "Follow me on Instagram",
```
Comment (krishguptadev) [inline public/data/itself13.json:17]:
```suggestion
        "name": "Find me on Instagram",
```

**Review by krishguptadev — CHANGES_REQUESTED** [review]
Looks good! But a few _suggestions_ from my side 🙂

File: public/data/itself13.json (hunk 1 above)
Comment (Pradumnasaraf) [reply public/data/itself13.json:22]: How this is wrong? Can you explain?

Comment (Pradumnasaraf) [reply public/data/itself13.json:22]: @krishguptadev How this is wrong? Can you explain?

File: public/data/itself13.json (hunk 3 above)
Comment (Vyvy-vi) [reply public/data/itself13.json:7]: This change is uneeded.

File: public/data/itself13.json (hunk 4 above)
Comment (Vyvy-vi) [reply public/data/itself13.json:17]: This change is also unneeded 😅

**Review by Vyvy-vi — COMMENTED** [review]
Looks good to me...
But, this does need some formatting.

(We should probably set up some tooling that automatically formats the JSON from PRs that are adding profiles)

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Comment by itself13** [comment]
Thanks broski

File: public/data/itself13.json [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+        "url": "https://www.instagram.com/ii_am_ayuuu/",
+        "icon": "Instagram"
+      }
+    ]
+}
```
Comment (krishguptadev) [inline public/data/itself13.json:22]:
```suggestion
}

```

Comment (Panquesito7) [inline public/data/itself13.json:22]: Missing an endline.
```suggestion
}

```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: public/data/itself13.json (hunk 5 above)
Comment (krishguptadev) [reply public/data/itself13.json:22]: Do not know why, but I requested the same changes: https://github.com/EddieHubCommunity/LinkFree/pull/677#discussion_r753069209 but this was resolved without commiting 🤔

**Review by Panquesito7 — APPROVED** [review]
LGTM 🚀 Thank you for your contribution! 😄👍

**Review by krishguptadev — APPROVED** [review]

**Pull request merged**